README.md

```diff
@@ -23,3 +23,31 @@ This repo builds the UDS Identity (Keycloak) Config image used by UDS Identity.
 ## Customizing UDS Identity Config
 
 If the default realm, plugin, theme, truststore, or jars do not provide enough functionality ( or provide too much functionality ), take a look at the [CUSTOMIZE.md](./docs/CUSTOMIZE.md) docs for making changes to the identity config.
+
+
+## Upgrading Identity Config
+
+### From v0.4.5 to v0.5.0
+
+This version upgrade brings in a new Authentication Flow for group authorization.
+
+If upgrading without a full redeploy of identity-config the following steps will be necessary to create and use group authorization:
```

Comment:
```suggestion
If upgrading without a full redeploy of keycloak the following steps will be necessary to create and use group authorization:
```

nit, but I think technically redeploying keycloak/recreating the realm is the key here?

Reply:
updated